geoips/create_plugin_registries.py

```diff
@@ -14,10 +14,128 @@
 import logging
 from geoips.commandline.log_setup import setup_logging
 import geoips.interfaces
+from geoips.errors import PluginRegistryError
 
 LOG = logging.getLogger(__name__)
 
 
+def remove_registries(plugin_packages):
+    """Remove all plugin registries if a PluginRegistryError is raised.
+
+    Parameters
+    ----------
+    plugin_packages: list EntryPoints
+        A list of EntryPoints pointing to each installed
+        GeoIPS package --> ie.
+        [EntryPoint(name='geoips', value='geoips', group='geoips.plugin_packages'), ...]
+    """
+    from os import remove
+
+    LOG.interactive(
+        "\n\n\n\nERROR: Removing registries due to duplicate plugins. You must fix the "
+        "error shown below before GeoIPS can operate correctly.\n\n\n"
+    )
+    for pkg in plugin_packages:
+        remove(str(resources.files(pkg.value) / "registered_plugins.yaml"))
```

Comment:
So this will remove all registries during install?  We will just need to make sure the install fails with a useful error message when we get to the point of tying this in with pyproject.toml?

Reply:
Yup! If a duplicate is found all registries are removed. I've added a few error messages that should point the user to fix those duplicates, but if you'd like me to modify them just let me know. These errors also state that you can't use GeoIPS functionality until these errors are fixed.

Reply:
As long as we make sure these error messages are raised appropriately when we integrate this with pyproject.toml, so if the installation fails it is obvious why, this is good with me.

Reply:
Looks good for now. We can handle the errors when we get to #385.